nautobot/core/graphql/generators.py

```diff
@@ -48,6 +48,21 @@ def resolve_custom_field(self, info, **kwargs):
     return resolve_custom_field
 
 
+def generate_computed_field_resolver(name, resolver_name):
+    """Generate function to resolve each computed field within each DjangoObjectType.
+
+    Args:
+        name (str): name of the computed field to resolve
+        resolver_name (str): name of the resolver as declare in DjangoObjectType
+    """
+
+    def resolve_computed_field(self, info, **kwargs):
+        return self.get_computed_field(name=name)
```

Comment:
Why is the first argument to this closure function called `self`? Could this be a little more descriptive like `obj` or `instance` so it's more clear on how this is being used by the caller?

Reply:
In this case, just being consistent in implementation of the other resolvers which all use `self` as the first argument. Not saying that you don't have a point though, cc: @dgarros